src/jit/lir.cpp

```diff
@@ -129,7 +129,14 @@ GenTree* LIR::Use::User() const
 //
 bool LIR::Use::IsValid() const
```

Comment:
Now that I look at this more closely, I think that the checks you've added are subsumed by the checks in the ctor: https://github.com/pgavlin/coreclr/pull/6/files#diff-8a1180cdfcc26882bd6daf7a4bb094d2L39